frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -842,7 +826,8 @@ export default ({
         const shouldLoadMoreEvents = messageHashToScroll && this.messages.findIndex(msg => msg.hash === messageHashToScroll) < 0
         if (shouldLoadMoreEvents) {
           const { height: latestHeight } = await sbp('chelonia/out/latestHEADInfo', chatRoomID)
-          if (!this.checkEventSourceConsistency(chatRoomID)) return
+
+          if (this.chatroomHasSwitchedFrom(chatRoomID)) return false
```

Comment:
Re-adding this check here fixes below error that occurs from time to time:

<img src='https://github.com/user-attachments/assets/a6619628-4b7d-4f10-a692-f0bbb253958d' width='480'>